_layouts/post.html

```diff
@@ -41,11 +41,18 @@ <h5>Author</h5>
         {%- endif -%}
         {%- if page.categories -%}
         <div class="post-tags info">
-            <ul>
-              {% for category in page.categories %}
-              <li>{{ category }}</li>
-              {% endfor %}
-          </ul>
+          <div class="field is-grouped is-grouped-multiline">
+            {% for category in site.categories %}
+              {% if page.categories contains category[0] %}
+                <div class="control">
+                  <div class="tags has-addons">
+                    <a href="/categories#{{ category[0] | downcase }}" class="tag is-medium is-primary">{{ category[0] }}</a>
```

Comment:
Is it necessary to create to <a> elements with the same `href` ?
I would propose to create one <a> element with to <span> because the two elements are only necessary for the visualisation (css classes), right?

Reply:
It's not my idea it was in the categories page already:

https://github.com/karakun/karakun.github.io/blob/7297eebfd233a6dbc8c984b8fc98e247099cdced/_layouts/categories.html#L10-L11

But I understand what you are saying if we do not want both a category name and a counter to have as a link element.